### ISSUE TITLE
Adding License

### ISSUE THREAD

**Issue by tyler-thetyrant** [acm-uiuc/display#8]
NCSA? MIT?

**Comment by kevinwang** [comment]
Re: your comment in #7, this project will be a client of the Groot API but not part of the Groot itself. So I think we're free to choose any license here. I'm partial to MIT because it's a well known license, particularly in the Node community.

**Comment by narendasan** [comment]
Yeah the choice of license does not need to be dependent of groot. I would request you add a license that has a trademark clause like BSD-3 Clause
